astro/src/content/docs/apis/_tenant-response-body-base.mdx

```diff
@@ -345,6 +345,14 @@ import JSON from 'src/components/JSON.astro';
     * `SlidingWindowWithMaximumLifetime` - the expiration is calculated from the last time the token was used, or until the <InlineField>maximumTimeToLiveInMinutes</InlineField> is reached. &nbsp; <AvailableSince since="1.46.0"/>
   </APIField>
 
+  <APIField name={ props.base_field_name + ".jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" } type="Integer" since="1.55.0">
```

Comment:
```suggestion
  <APIField name={ props.base_field_name + ".jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" } type="Integer" since="1.55.1">
```